pkg/archive/archive.go

```diff
@@ -405,6 +405,8 @@ func Tar(path string, compression Compression) (io.ReadCloser, error) {
 // paths are included in `options.IncludeFiles` (if non-nil) or not in `options.ExcludePatterns`.
 func TarWithOptions(srcPath string, options *TarOptions) (io.ReadCloser, error) {
 
+	srcPath = fixVolumePathPrefix(srcPath)
```

Comment:
Kind of. I would think most Windows developers will be familiar with the \\\\?\\ type usage. I know it's unusual for those with a Linux background new to Windows though.